core/src/main/java/co/aikar/commands/BaseCommand.java

```diff
@@ -329,43 +532,19 @@ private void registerSubcommand(Method method, String subCommand) {
     }
 
     /**
-     * Takes a string like "foo|bar baz|qux" and generates a list of
-     * - foo baz
-     * - foo qux
-     * - bar baz
-     * - bar qux
+     * Executes this command properly by handling it through other methods (look at the see fields).
```

Comment:
def need better wording.

This is the primary entry point for integrations to take raw input and map it to a command to execute.